src/wlapplication_messages.cc

```diff
@@ -208,6 +208,10 @@ void fill_parameter_vector() {
 		true},
 	  /// Others
 	  {_("Others:"), "verbose", "", _("Enable verbose debug messages"), false},
+	  {"", "verbose-i18n", "",
+		_("Print all strings as they are translated. This helps with tracing down bugs with "
+		  "internationalization."),
+		false},
```

Comment:
```suggestion
		true},
```

Reply:
This would make it default to true, not sure if this is wise?

Reply:
This is not a default value, this boolean only decides whether the option is listed with --help (false) or only with --help-all (true)